end_to_end_testing/setup.sh

```diff
@@ -12,25 +12,25 @@ sudo cp /tmp/binaries/reckoner-linux-amd64 /usr/local/bin/reckoner
 reckoner version
 
 echo "Installing Kind"
-curl -sLO https://github.com/kubernetes-sigs/kind/releases/download/0.2.1/kind-linux-amd64
+curl -sLO https://github.com/kubernetes-sigs/kind/releases/download/0.4.0/kind-linux-amd64
 chmod 0755 kind-linux-amd64
 sudo mv kind-linux-amd64 /usr/local/bin/kind
 kind version
 
 echo "Installing Kubectl"
-curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.12.7/bin/linux/amd64/kubectl
+curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.14.4/bin/linux/amd64/kubectl
 chmod 0755 kubectl
 sudo mv kubectl /usr/local/bin/
 kubectl version --client
 
 echo "Installing Helm"
-curl -sL https://storage.googleapis.com/kubernetes-helm/helm-v2.13.1-linux-amd64.tar.gz | tar xzv linux-amd64/helm
+curl -sL https://storage.googleapis.com/kubernetes-helm/helm-v2.14.2-linux-amd64.tar.gz | tar xzv linux-amd64/helm
 sudo mv linux-amd64/helm /usr/local/bin/helm
 rm -rf linux-amd64
 helm version --client
 
 echo "Creating Kubernetes Cluster with Kind"
-kind create cluster --wait=90s
+kind create cluster --wait=90s --image kindest/node:v1.14.3
```

Comment:
This was the latest available release from kind (for v1.14) even though Kubernetes is on 1.14.4